.frontmatter/database/mediaDb.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
What is this file?

Reply:
removed, came from my vscode extension!

Reply:
I always recommend reviewing a PR yourself before submitting it for review by others.